nifi-frontend/src/main/frontend/apps/nifi/src/app/ui/common/extension-creation/extension-creation.component.html

```diff
@@ -15,23 +15,26 @@
   ~ limitations under the License.
   -->
 <div class="extension-creation-dialog" (keyup)="navigateSelectionList($event)">
-    <div class="flex justify-between items-center">
-        <h2 mat-dialog-title>Add {{ componentType }}</h2>
-        <div class="pt-5 pr-5">
-            <mat-form-field>
-                <mat-label>Filter types</mat-label>
-                <input
-                    matInput
-                    class="px-1.5 py-1 w-64"
-                    type="text"
-                    (keyup)="filterTypes($event)"
-                    tabindex="0"
-                    placeholder="Filter types..." />
-            </mat-form-field>
-        </div>
-    </div>
+    <h2 mat-dialog-title>Add {{ componentType }}</h2>
     <mat-dialog-content>
-        <div class="flex flex-col dialog-content">
+        <div class="flex flex-col dialog-content gap-y-2">
+            <div class="flex flex-col">
+                <div class="flex items-center gap-x-2">
```

Comment:
```suggestion
                <div class="flex items-center">
```

There is only a single child element so the `gap` isn't needed here.